mms_message_test.go

```diff
@@ -0,0 +1,165 @@
+package messagebird
+
+import (
+	"testing"
+	"time"
+)
+
+var mmsMessageObject []byte = []byte(`{
```

Comment:
You don't need to specify the type of the variable when it can be deduced from the value. So here you can write `var mmsMessageObject = []byte(`.